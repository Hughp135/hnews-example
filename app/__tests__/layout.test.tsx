import { render } from "@testing-library/react";
import RootLayout from "../layout";

describe("Page", () => {
  it("renders the layout", () => {
    const { asFragment } = render(
      <RootLayout>
        <p>Content</p>
      </RootLayout>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
