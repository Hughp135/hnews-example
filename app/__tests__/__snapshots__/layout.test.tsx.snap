// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders the layout 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <img
      alt="Hacker News Top 100"
      class="bgImage"
      data-nimg="fill"
      decoding="async"
      loading="lazy"
      sizes="100vw"
      src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
      style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
    />
    <h1>
      Hacker News Top 100 Stories
    </h1>
  </header>
  <p>
    Content
  </p>
</DocumentFragment>
`;
